tests/phpunit/CRM/Contact/Import/Form/MapFieldTest.php

```diff
@@ -61,9 +61,9 @@ public function tearDown(): void {
    * @throws \CiviCRM_API3_Exception
    */
   public function testSubmit(array $params, array $mapper, array $expecteds = []): void {
-    $form = $this->getMapFieldFormObject(['mapper' => $mapper]);
+    $form = $this->getMapFieldFormObject(array_merge($params, ['mapper' => $mapper]));
     $form->preProcess();
-    $form->submit($params);
+    $form->postProcess();
```

Comment:
This is a change that is required for the bigger PR rather than the small one - but this `submit` function is on the way out since we now have a harness that tests the `postProcess` directly